prime-router/metadata/hl7_mapping/common/software.yml

```diff
@@ -1,37 +1,9 @@
 elements:
-  - name: software-vendor-name
-    value: [ '%resource.source.extension(%`rsext-software-vendor-org`).value.resolve().name' ]
-    hl7Spec: [ SFT-1-1 ]
-
-  - name: software-vendor-name-type
-    # software-vendor-org is a reference
-    resource: '%resource.source.extension(%`rsext-software-vendor-org`).value.resolve()'
-    condition: '%resource.name.exists()'
-    value:
-      - '%resource.extension(%`rsext-organization-name-type`).value.coding.code'
-    hl7Spec: [ SFT-1-2 ]
-
```

Comment:
Why were these removed?

Reply:
They were removed to go through `xon-organization` for all of SFT-1 instead of doing it piecemeal for each SFT-1 subfield individually. In the base case, this is a lot cleaner, although it makes it a little more complicated to override specific subsegments of SFT-1 (as we do in California). For instance, since we can't just override `software-vendor-name-type` with `condition: false` anymore, the CA receiver transform instead has to create a new element that will sit at the root of the schema that can clear out SFT.1.2 .